lib/block_view_message.go

```diff
@@ -825,6 +856,76 @@ func (bav *UtxoView) _connectMessagingGroup(
 		}
 	}
 
+	// V3 Messages: MUTING and UNMUTING members
+	if blockHeight >= bav.Params.ForkHeights.DeSoV3MessagesMutingBlockHeight {
+		if existingEntry != nil && !existingEntry.isDeleted {
+			value, operationTypeExists := txn.ExtraData["OperationType"]
+			if operationTypeExists {
+				// inline helper method
+				contains := func(memberList []*MessagingGroupMember, member *MessagingGroupMember) bool {
+					for _, a := range memberList {
+						if a == member {
+							return true
+						}
+					}
+					return false
+				}
+				// make deep copy of existingEntry.MuteList to prevent mempool errors
+				muteListCopy := make([]*MessagingGroupMember, len(existingEntry.MuteList))
+				copy(muteListCopy, existingEntry.MuteList)
+				// check if mute or unmute
+				if string(value) == MessagingGroupOperationMute {
```

Comment:
I wonder if we want to throw an error whenever the member was already in the mutelist or if they weren't muted, wyt?

Reply:
yes, done